src/share/util/shr_abort_mod.F90

```diff
@@ -0,0 +1,155 @@
+module shr_abort_mod
```

Comment:
Please add a note here that components should not use this module directly and should continue to use shr_sys_mod: shr_sys_abort. I think that it's important to maintain a single method in the API to access this function.